index.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel='icon' href='./Bilder/favicon.png' type='image/x-icon'/>
+    <title>noemi's vinyls</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Peddana&family=Permanent+Marker&display=swap" rel="stylesheet"> 
+    <style> html, body{ 
+      scroll-behavior: smooth;
+      }</style>
+    <!-- dont forget the meta tag for responsiveness -->
+    </head>
+
+  <body>
+    <div class="title-area">
+      <div class="introduction"><h1>noemi's vinyl collection</h1>
+        <p class="introduction">finally i got around to inventory my vinyls! 
+        the main-purpose here is to keep myself from buying the same vinyl twice. seems simple, but happened already twice.</p>
+        </div>
+      </div>
+    
+
+
+      <p class="alphabet">
+            <a href="#a.">a.</a>
+            <a href="#b.">b.</a>
+            <a href="#c.">c.</a>
+            <a href="#d.">d.</a>
+            <a href="#e.">e.</a>
+            <a href="#f.">f.</a>
+            <a href="#g.">g.</a>
+            <a href="#h.">h.</a>
+            <a href="#i.">i.</a>
+            <a href="#j.">j.</a>
+            <a href="#k.">k.</a>
+            <a href="#l.">l.</a>
+            <a href="#m.">m.</a>
+            <a href="#n.">n.</a>
+            <a href="#o.">o.</a>
+            <a href="#p.">p.</a> 
+            <a href="#q.">q.</a> 
+            <a href="#r.">r.</a>
+            <a href="#s.">s.</a>
+            <a href="#t.">t.</a> 
+            <a href="#u.">u.</a>
+            <a href="#v.">v.</a>
+            <a href="#w.">w.</a>
+            <a href="#x.">x.</a>
+            <a href="#y.">y.</a> 
+            <a href="#z.">z.</a>
+            <a href="##.">#.</a>
+          </p>
+
+
+
+    <main class="vinyls">
+      <section class="flex-parent">
+      <!-- empty code:  <div class="flex-child "><h2></h2><img src="" alt="" width="250px"></div> -->
+      <div class="flex-child"><h3 id="a.">a.</h3></div>
+      <div class="flex-child"><h2>allan kingdom - lines</h2> <img src="Bilder/allan kingdom - lines.JPG" alt="allan kingdom" width="250px"></div>
+      <div class="flex-child"><h2>anderson .paak - malibu</h2> <img src="./Bilder/anderson .paak - malibu.JPG" alt="anderson .paak" width="250px"></div>
+      <div class="flex-child"><h2>anderson .paak - o.b.e</h2><img src="./Bilder/anderson .paak - o.b.e. (out of body experience).JPG" alt="anderson .paak" width="250px"></div>
+      <div class="flex-child"><h2>anderson .paak - venice</h2><img src="./Bilder/anderson .paak - venice.JPG" alt="anderson .paak" width="250px"></div>
```

Comment:
Inside this whole section (`flex-parent`), we should add some indentation and line breaks to make it easier to read. Also - keep the styling to the CSS file.

```
<div class="flex-child">
  <h3 id="a.">a.</h3>
</div>
<div class="flex-child">
  <h2>allan kingdom - lines</h2>
  <img src="Bilder/allan kingdom - lines.JPG" alt="allan kingdom" width="250px">
</div>
```
etc